tests/testing/LocalBearTestHelperTest.py

```diff
@@ -78,19 +78,12 @@ def test_bad_assertComparableObjectsEqual(self):
                                     file=fname)],
                                    filename=fname,
                                    create_tempfile=False)
-        self.assertEqual('\'LineCountTestBear\' != \'LineCountTestBea\'\n'
-                         '- LineCountTestBear\n'
-                         '?                 -\n'
-                         '+ LineCountTestBea\n'
-                         ' : origin mismatch.\n\n'
-                         '\'This file has 3 lines.\' != \'This file has 2 '
-                         'lines.\'\n'
-                         '- This file has 3 lines.\n'
-                         '?               ^\n'
-                         '+ This file has 2 lines.\n'
-                         '?               ^\n'
-                         ' : message_base mismatch.\n\n',
-                         str(cm.exception))
+        self.assertEqual('origin mismatch: LineCountTestBear, This file '
+                         'has 3 lines. != LineCountTestBea, This file '
+                         'has 2 lines.\n\n'
+                         'message_base mismatch: LineCountTestBear, This '
```

Comment:
this is a bug somewhere, and a regression.

1. origin mismatch & message_base mismatch both say the same thing. that is confusing.  something needs to be improved.

2. the pre-existing shows the line differences, and your change doesnt.  That is a regression, making it more difficult for bear writers to see the problem, not more easy which your project was supposed to do.

Reply:
Done in PR https://github.com/coala/coala/pull/5564